kolibri/plugins/learn/assets/src/actions.js

```diff
@@ -311,23 +310,26 @@ function showLearnChannel(store, channelId) {
     const popularPromise = ContentNodeResource.getCollection(popularPayload).fetch();
     const resumePromise = ContentNodeResource.getCollection(resumePayload).fetch();
     _updateChannelList(store);
-    ConditionalPromise.all([nextStepsPromise, popularPromise, resumePromise])
```

Comment:
Lint that code block!

(Yeah, I realize this probably happened while you were changing the name to `Generator` which seems reasonable, so as not to obfuscate the nature of the function).